.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
He is a good idea to have a version of the IntelliJ configurations?

Reply:
Should we remove this file then? 😰

Reply:
No, keep the changes